src/main/java/frc/utils/ctre/BusChain.java

```diff
@@ -5,58 +5,73 @@
 import frc.robot.constants.LogPaths;
 import org.littletonrobotics.junction.Logger;
 
+
 public enum BusChain {
 
 	ROBORIO("rio"),
```

Comment:
convention is log path folders begin with uppercase

Reply:
but it does start with uppercase

Reply:
when logging the roborio CANbus chain the log path would be Bus/rio/... and not Bus/Rio/...

Reply:
ok, fixed